### PR TITLE
Fix/liquidity icon

### PR DESCRIPTION
**Pull request by saurabh391**
#### What kind of PR is this?:
UI: Add liquidity icon #537

/kind fix

#### What this PR does / why we need it:
To address the liquidity icons issue.

#### Which issue(s) does this PR fixes?:
UI: Add liquidity icon #537

Fixes #

### CODE REVIEW

**Review by monstrobishi — APPROVED** [review]

File: webapp/src/containers/WalletPage/components/WalletTxnsNew/index.tsx [hunk 1]
```diff
@@ -296,6 +296,12 @@ const WalletTxns: React.FunctionComponent<WalletTxnsProps> = (
     if (type === POOL_SWAP_CATEGORY_LABEL) {
       return <MdCompareArrows className={styles.typeIcon} />;
     }
+    if ([REMOVE_LIQUIDITY_LABEL].includes(type)) {
+      return <MdArrowUpward className={styles.typeIcon} />;
```
Comment (thedoublejay) [inline webapp/src/containers/WalletPage/components/WalletTxnsNew/index.tsx:300]: REMOVE_LIQUIDITY_LABEL is already defined above. I think we need to clarify this ticket

**Review by thedoublejay — APPROVED** [review]

**Comment by monstrobishi** [comment]
@saurabh391 I think it's backwards, remove should point down, add should point up

File: webapp/src/containers/WalletPage/components/WalletTxnsNew/index.tsx (hunk 1 above)
Comment (thedoublejay) [reply webapp/src/containers/WalletPage/components/WalletTxnsNew/index.tsx:300]: update

**Review by thedoublejay — CHANGES_REQUESTED** [review]

**Comment by monstrobishi** [comment]
Actually it's already handled correctly, this change reverses the arrows which not what we want

**Pull request closed without merging**